### ISSUE TITLE
Fees displayed on SwapModal in case set spending cap is involved

### ISSUE THREAD

**Issue by Khushboo-dev-cpp** [status-im/status-desktop#15439]
We need to think about how we display fees on the SwapModal when setting a spending cap is required first https://www.figma.com/design/TS0eQX9dAZXqZtELiwKIoK/Swap---Milestone-1?node-id=3531-444171&m=dev

currently out implementation only shows approvals fees during approval flow and after approval is completed the fees needed for actual swap.

The user never sees how the much it will cost him in total to do the swap and maybe we should think about this.

this discussion started here https://github.com/status-im/status-desktop/pull/15411#discussion_r1662962625

@benjthayer @xAlisher @dlipicar

**Comment by benjthayer** [comment]
> We need to think about how we display fees on the SwapModal when setting a spending cap is required first https://www.figma.com/design/TS0eQX9dAZXqZtELiwKIoK/Swap---Milestone-1?node-id=3531-444171&m=dev
> 
> currently out implementation only shows approvals fees during approval flow and after approval is completed the fees needed for actual swap.
> 
> The user never sees how the much it will cost him in total to do the swap and maybe we should think about this.
> 
> this discussion started here [#15411 (comment)](https://github.com/status-im/status-desktop/pull/15411#discussion_r1662962625)
> 
> @benjthayer @xAlisher @dlipicar

Good point @Khushboo-dev-cpp - perhaps in this state we use `Approval fees:` instead of `Max fees:`? WDYT @xAlisher - just trying to think of a minimal change that might make things clearer.
